src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1501,7 +1503,9 @@
 	function m.platformToolset(cfg)
 		local tool, version = p.config.toolset(cfg)
 		if version then
-			version = "v" .. version
+			if tonumber(version:sub(1,3)) ~= nil then
+				version = "v" .. version
```

Comment:
No, I'm not. Just ended up that way because of the order in which the feature got implemented.